message GameStateMessage{
	required uint32 player_number = 6 [default = 1];
	required uint32 team_number = 1 [default = 9];
	required uint32 team_color =7 [default =0];
	required uint32 own_goal_color =5 [default =0];
	required uint32 player_state = 8 [default =0];
	required uint32 game_state = 9 [ default =0];
	required uint32 sec_game_state = 10 [ default =0];
	required bool firstHalf = 3 [default = true];
	required uint32 penalty = 13 [default = 0];
	required bool kickoff = 12 [default = true];
	required uint32 our_score = 14 [default = 0];
	required uint32 otherteam_score = 15 [default = 0];
	required uint32 override_state=16 [default = 0];
	required uint32 previous_player_state=17 [default = 0];
}

message LedValues{
	required string chain = 1 [default = ""];
	required string color = 2 [default = "off"];

}
message LedChangeMessage{
	repeated LedValues leds =6;
}

message ConfigMessage{
	required int32 team_number = 1 [default = 9];
	required int32 player_number = 2 [default = 1];
	required int32 team_color = 4 [default = 0];
	required int32 port =  3 [default = 3838];
}
